pyproject.toml

```diff
@@ -203,13 +203,16 @@ warn_unused_ignores = false
 
 [[tool.mypy.overrides]]
 # # TODO: this should be changed to true after a transition period
```

Comment:
```suggestion
```
Another option could be to set this option true in the global settings and the override the subpackages which still do not work (`cartesian` and a couple of modules in `next`, I guess)

Reply:
this small comment cost me 3h, because I realized that so far I didn't enable `disallow_untyped_defs`...